tamr_toolbox/utils/version.py

```diff
@@ -49,8 +53,128 @@ def _as_float(version: str) -> float:
     return (version_split[0] * 1000) + version_split[1] + (version_split[2] / 10)
 
 
+def _get_tamr_versions_from_function_args(*args, **kwargs) -> List[str]:
+    all_args = locals()
+    args = [arg for arg in all_args["args"]]
+    kwargs = list(all_args["kwargs"].values())
+    all_args_parsed = args + kwargs
+    response = []
+
+    # Return the client version (if we can find it)
+    for arg in all_args_parsed:
+        if type(arg) is Client:
+            response.append(current(arg))
+        elif hasattr(arg, "client"):
+            if type(arg.client) is Client:
+                response.append(current(arg.client))
+
+    return response
+
+
+def is_version_condition_met(
+    tamr_version: str,
+    min_version: str,
+    max_version: Optional[str] = None,
```

Comment:
Let's use the same default `None` for both, so this function can be used to enforce only a maximum, in case any Tamr features end up deprecated.  This might mean you need slightly more logic in constructing the error message.
```suggestion
    min_version: Optional[str] = None,
    max_version: Optional[str] = None,
```

Reply:
I think this is maybe one we discuss - my concern is that otherwise the function could be called completely empty, and by requiring a minimum we enforce good usage of the code.